index.html

```diff
@@ -0,0 +1,536 @@
+        class="max-w-container px-4 mx-auto py-24 md:py-36 skew-y-6 grid md:grid-cols-4 gap-10"
+      >
+        <div class="col-span-2">
+          <p class="text-lg font-bold text-blue-400 mb-3">
+            Designed for developers
+          </p>
+          <h1 class="text-4xl font-bold text-white mb-3">
+            The world’s most powerful and easy-to-use APIs
+          </h1>
+          <p class="font-medium text-base text-gray-400 mb-5">
+            We agonize over the right abstractions so your teams don’t need to
+            stitch together disparate systems or spend months integrating
+            payments functionality.
+          </p>
+          <button
+            class="bg-blue-400 rounded-full text-white text-sm px-6 py-2 hover:bg-white hover:text-slate-800"
+          >
+            Read the docs
+          </button>
+        </div>
+        <!-- simple div 1  -->
+        <div class="md:col-start-1 md:-mt-28">
+          <img class="w-10" src="assets/terminal_icon.png" alt="" />
+          <p class="font-bold text-white border-l-2 border-blue-400 pl-2 my-3">
+            Tools for every stack
+          </p>
+          <p class="text-gray-400 text-sm mb-2">
+            We offer client and server libraries in everything from React and
+            PHP to .NET and iOS
+          </p>
+          <button class="text-blue-300 text-sm font-bold hover:text-white">
+            See libraries
+          </button>
+        </div>
+
+        <!-- simple div 2  -->
+        <div class="col-start-1 md:col-start-2 md:-mt-28">
+          <img class="w-10" src="assets/boxes.png" alt="" />
+          <p class="font-bold text-white border-l-2 border-blue-400 pl-2 my-3">
+            Prebuilt integrations
+          </p>
+          <p class="text-gray-400 text-sm mb-3">
+            Use integrations for systems like Shopify, WooCommerce, NetSuite,
+            and more.
+          </p>
+          <button class="text-blue-300 text-sm font-bold hover:text-white">
+            See libraries
+          </button>
+        </div>
+        <!-- Terminal Dashboard  -->
+        <img
+          class="col-start-1 col-span-2 w-full px-4 md:row-start-1 md:row-span-2 md:col-start-3 md:col-span-2"
+          src="assets/terminal_dashboard.png"
+          alt=""
+        />
+      </div>
+    </div>
+
+    <!-- starting of why stripe  -->
+
+    <div class="max-w-container px-4 mx-auto mt-20 pb-20">
+      <p class="text-lg font-bold text-blue-400 mb-3">Why Stripe</p>
+      <h1 class="text-4xl font-bold text-slate-800 mb-5">
+        A technology-first approach to payments <br />
+        and finance
+      </h1>
+
+      <div class="grid md:grid-cols-4 gap-10 my-20">
+        <!-- item 1 -->
+        <div>
+          <img class="w-10" src="assets/close.png" alt="" />
+          <p class="font-bold text-sm my-4 border-l border-l-indigo-700 pl-3">
+            Close to the metal
+          </p>
+          <p class="text-sm text-gray-500 leading-relaxed">
```

Comment:
the original one
<img width="1440" alt="Screen Shot 2022-11-22 at 09 20 09" src="https://user-images.githubusercontent.com/94446528/203250234-07d935cd-7d04-43ce-85fe-fef0103419cc.png">

Yours
<img width="1440" alt="Screen Shot 2022-11-22 at 09 19 54" src="https://user-images.githubusercontent.com/94446528/203250346-a5f45efd-9f00-466e-9c75-8a2238948a0f.png">


Try to add those colors to those words and alss they are hovered